Front-End/challenge2/script.js

```diff
@@ -5,3 +5,171 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+
+let a;
```

Comment:
for next challenges, please try to use more descriptive variables names,  because if you add more and more code later naming a variable like `a` or `b` would make the code confusing also I'd suggest to initialize the variables so that way we know which type of value you are expecting to assign...this problems get resolved with typescript later but please take it in count